src/content/energy-consumption/index.md

```diff
@@ -4,65 +4,66 @@ description: The basic information you need to understand Ethereum's energy cons
+While Ethereum's energy consumption is very low, there is also a substantial, growing, and highly active **regenerative finance (ReFi)** community building on Ethereum. ReFi applications use DeFi components to build financial applications that have positive externalities benefiting the environment. ReFi is part of a wider ["solarpunk"](https://en.wikipedia.org/wiki/Solarpunk) movement that is closely aligned with Ethereum and aims to couple technological advancement and environmental stewardship. The decentralized, permissionless, composable nature of Ethereum makes it the ideal base layer for the ReFi and solarpunk communities.
 
-Read more on [how Ethereum implements proof-of-stake](/developers/docs/consensus-mechanisms/pos) and how it compares to proof-of-work.
+Web3 native public goods funding platforms such as [Gitcoin](gitcoin.co) run climate rounds specifically in order to stimulate environmentally conscious building on Ethereum's application layer. Through the development of these initiatives (and others, e.g. [DeSci](/desci/)), Ethereum is becoming an environmentally and socially net-positive technology.
 
 <InfoBanner emoji=":evergreen_tree:">
-  If you think these stats are incorrect or can be made more accurate, please raise an issue or PR. These are estimates by the ethereum.org team made using publicly accessible information and the current Ethereum roadmap. These statements don't represent an official promise from the Ethereum Foundation. 
+  If you think this page can be made more accurate, please raise an issue or PR. The stats on this page are estimates based on publicly available data - they do not represent an official statement or promise from the Ethereum Foundation. 
 </InfoBanner>
 
 ## Further reading {#further-reading}
 
+- [White House report on PoW blockchains](https://www.whitehouse.gov/wp-content/uploads/2022/09/09-2022-Crypto-Assets-and-Climate-Report.pdf)
 - [A country's worth of power, no more](https://blog.ethereum.org/2021/05/18/country-power-no-more/) – _Carl Beekhuizen, May 18 2021_
```

Comment:
Should we remove this?

Reply:
Sorry @wackerow, this comment was regarding Carl's article. Not a strong take, but it doesn't really seem as relevant anymore (if we're trying to communicate how Ethereum consumes energy today versus how it did in the past)

Reply:
yea it is a very old blog post, included because it was the first estimate I'm aware of, and most other articles have built on it in some way. But, it can be removed, probably better tbh.
```suggestion
```